exampleSite/static/admin/config.yml

```diff
@@ -184,9 +186,14 @@ collections:
           widget: "select"
           options: ["tanks"]
           default: ["tanks"]
+        - {label: "Layout", name: "layout", widget: "select", options: ["role_home"], default: ["role_home"]}
+        - {label: "Background Header Image", name: "bg_header_image", widget: "image", choose_url: false}
+        - {label: "Role Hero Image", name: "role_hero_image", widget: "image", choose_url: false}
       - label: "Healers"
```

Comment:
In each of the roles I've added

- Media Folder path
- Public Folder path
- Dropdown to select the role_home layout. It is the only one available to guarantee.
- Background Header Image
- Hero Image

The role icon with the border should already be in balance-static